controllers/imagebasedgroupupgrade.go

```diff
@@ -305,13 +299,8 @@ func (r *IBGUReconciler) ensureCGUForPlanItem(
 			r.Log.Info("ManifestWorkReplicaSet already exist", "ManifestWorkReplicaSet", mwrs.Name)
 		}
 	}
-	annotations := make(map[string]string)
```

Comment:
Please keep this and the function argument down below. I used it for passing the kuttl suffix annotation to CGU